### PR TITLE
fix blockParticleEffect message

### LINKED ISSUE #1384: Baker update - Ray issues
Baker is working after restart but client crashed!
Minecraft version 1.10.2
minecolonies-universal-1.10.2-0.8.3524

[latest.zip](https://github.com/Minecolonies/minecolonies/files/1240139/latest.zip)
scroll down for the crash :)

Many greetings
~H3ro

### PR DESCRIPTION
**Pull request by Raycoms**
Closes #1384 

# Changes proposed in this pull request:
- Adapts blockParticleEffectMessage
* To work on the server thread
* To use the correct side

Review please

### CODE REVIEW

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 56 issues
* ![MAJOR][MAJOR] 21 major
* ![MINOR][MINOR] 34 minor
* ![INFO][INFO] 1 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [EntityCitizen.java](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java): This file has 1.320 lines, which is greater than 1.000 authorized. Split it into smaller files. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00104)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 57 to the maximum authorized 30 or less. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1200)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): class "EntityCitizen" has 95 methods, which is greater than the 35 authorized. Split it into smaller classes. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L359](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L359): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L361](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L361): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L362](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L362): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L363](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L363): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Assign this magic number 3.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L366](https://github.com/Minecolonies/minecolonies/blob/b94cd55c3bad39105f41124eb4e10ff5997c3bb3/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L366): Assign this magic number 5.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/com/minecolonies/coremod/network/messages/BlockParticleEffectMessage.java [hunk 1]
```diff
@@ -70,18 +71,15 @@ public void toBytes(@NotNull final ByteBuf buf)
         buf.writeInt(side);
     }
 
-    @Nullable
```
Comment (cltnschlosser) [inline src/main/java/com/minecolonies/coremod/network/messages/BlockParticleEffectMessage.java]: You can add this back

File: src/main/java/com/minecolonies/coremod/network/messages/BlockParticleEffectMessage.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import net.minecraft.block.Block;
 import net.minecraft.block.state.IBlockState;
 import net.minecraft.client.Minecraft;
+import net.minecraft.entity.player.EntityPlayerMP;
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/network/messages/BlockParticleEffectMessage.java]: MINOR Remove this unused import 'net.minecraft.entity.player.EntityPlayerMP'. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 55 issues
* ![MAJOR][MAJOR] 21 major
* ![MINOR][MINOR] 33 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [EntityCitizen.java](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java): This file has 1.320 lines, which is greater than 1.000 authorized. Split it into smaller files. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00104)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 57 to the maximum authorized 30 or less. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1200)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): class "EntityCitizen" has 95 methods, which is greater than the 35 authorized. Split it into smaller classes. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L359](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L359): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L361](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L361): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L362](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L362): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L363](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L363): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Assign this magic number 3.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L366](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L366): Assign this magic number 5.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by cltnschlosser — APPROVED** [review]

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 56 issues
* ![MAJOR][MAJOR] 21 major
* ![MINOR][MINOR] 34 minor
* ![INFO][INFO] 1 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [EntityCitizen.java](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java): This file has 1.320 lines, which is greater than 1.000 authorized. Split it into smaller files. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00104)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 57 to the maximum authorized 30 or less. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1200)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): class "EntityCitizen" has 95 methods, which is greater than the 35 authorized. Split it into smaller classes. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L359](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L359): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L361](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L361): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L362](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L362): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L363](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L363): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Assign this magic number 3.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L366](https://github.com/Minecolonies/minecolonies/blob/63e58589f1cfa3661e555b7c0161943350a6cbd8/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L366): Assign this magic number 5.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by marchermans — APPROVED** [review]

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 55 issues
* ![MAJOR][MAJOR] 21 major
* ![MINOR][MINOR] 33 minor
* ![INFO][INFO] 1 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [EntityCitizen.java](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java): This file has 1.320 lines, which is greater than 1.000 authorized. Split it into smaller files. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00104)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 57 to the maximum authorized 30 or less. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1200)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): class "EntityCitizen" has 95 methods, which is greater than the 35 authorized. Split it into smaller classes. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L359](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L359): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L361](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L361): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L362](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L362): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L363](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L363): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Assign this magic number 3.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L366](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L366): Assign this magic number 5.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 53 issues
* ![MAJOR][MAJOR] 21 major
* ![MINOR][MINOR] 31 minor
* ![INFO][INFO] 1 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [EntityCitizen.java](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java): This file has 1.320 lines, which is greater than 1.000 authorized. Split it into smaller files. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00104)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 57 to the maximum authorized 30 or less. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1200)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L73](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L73): class "EntityCitizen" has 95 methods, which is greater than the 35 authorized. Split it into smaller classes. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L359](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L359): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L361](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L361): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L362](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L362): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L363](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L363): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Assign this magic number 3.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L365](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L365): Extract this increment or decrement operator into a dedicated statement. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS881)
1. ![MAJOR][MAJOR] [EntityCitizen.java#L366](https://github.com/Minecolonies/minecolonies/blob/3e410b555ad06846e20b612c61ef5815476fd5a5/src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java#L366): Assign this magic number 5.0F to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**